gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -576,21 +597,52 @@ def get_combine_faf_resources(
             },
         },
     )
-    contingency_table_test = PipelineStepResourceCollection(
-        "--perform-contingency-table-test",
-        output_resources={
+
+    cmh_resources = {}
+    contingency_test_resources = {}
+    if stats_chr is not None:
+        cmh_resources["output_resources"] = {
+            "cmh_ht": TableResource(get_checkpoint_path(f"cmh_{stats_chr}"))
+        }
+        contingency_test_resources["output_resources"] = {
+            "contingency_table_ht": TableResource(
+                get_checkpoint_path(f"contingency_table_{stats_chr}")
+            )
+        }
+    else:
+        cmh_resources["output_resources"] = {
+            "cmh_ht": get_freq_comparison("cmh_test", test=test, filtered=filtered)
+        }
+        contingency_test_resources["output_resources"] = {
             "contingency_table_ht": get_freq_comparison(
                 "contingency_table_test", test=test, filtered=filtered
             )
-        },
-        pipeline_input_steps=[combined_frequency],
+        }
+
+    if stats_combine_all_chr:
+        cmh_resources["input_resources"] = {
+            "--perform-cochran-mantel-haenszel-test --stats-chr (all chr)": {
```

Comment:
Small thing but maybe 
```suggestion
            "--perform-cochran-mantel-haenszel-test --stats-chr (any chr)": {
```
At first pass, this was a bit confusing as I though it was suggesting a list of all chrs